lib/parsers/parse5.js

```diff
@@ -30,3 +30,22 @@ exports.render = function (dom) {
     { treeAdapter: htmlparser2Adapter }
   );
 };
+
+// Attempts to match single html elements
+// like <..></..>, <../>, <..>
+var rsingleTag = /^<([a-z][^/\0>:\x20\t\r\n\f]*)[\x20\t\r\n\f]*\/?>(?:<\/\1>)?$/i;
+// HTML namespace
+var ns_html = 'http://www.w3.org/1999/xhtml';
```

Comment:
Shouldn't this be `nsHtml`? Or even better drop the var since it's used once?

Reply:
I put it in var because prettier formatting in sentence looked awkward, but naming you are right.